testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -105,20 +144,16 @@ def different_atoms():
 
         assert_raises(SelectionError, different_atoms)
 
+
 class TestAlignmentProcessing(TestCase):
     def setUp(self):
         self.seq = FASTA
-        fd, self.alnfile = tempfile.mkstemp(suffix=".aln")
-        os.close(fd)
-        fd, self.treefile = tempfile.mkstemp(suffix=".dnd")
-        os.close(fd)
+        self.tempdir = tempdir.TempDir()
+        self.alnfile = self.tempdir.name + '/alignmentprocessing.aln'
```

Comment:
generally, use `os.path.join()` instead of string concatenation
```python
import os.path
self.alnfile = os.path.join(self.tempdir.name,  'alignmentprocessing.aln')
```

so that you don't have to worry about path separators (and it should work on Windows, too, ... eventually).